lodash/index.d.ts

```diff
@@ -12335,7 +12335,7 @@ declare module _ {
          * @param value The value to check.
          * @return Returns true if value is null, else false.
          */
-        isNull(value?: any): boolean;
+        isNull(value?: any): value is null;
```

Comment:
Ok I amended my commit and pushed it. Tests surprisingly failed. Looking at the logs it seemed unrelated with my changes so rolled back and pushed again. Tests still failed whereas they were working for the exact same code. Something must have broken in the branch.
I'll try again when the main branch will be stable again.